test_scripts/Polices/External_UCS/External_Consent_ON/013_ATF_Policies_External_Consent_ON_through_ignition_cycles_disallowed_by_external_consent_entities_omitted.lua

```diff
@@ -0,0 +1,163 @@
+-------------------------------------- Requirement summary -------------------------------------------
+-- [Policies] External UCS: "ON" status between ignition cycles
+-- [Policies] External UCS: "ON" in case of omitted "disallowed_by_external_consent_entities" param in functional groupings
+--
+------------------------------------------------------------------------------------------------------
+------------------------------------ General Settings for Configuration ------------------------------
+------------------------------------------------------------------------------------------------------
+require('user_modules/all_common_modules')
+local common_functions_external_consent = require('user_modules/shared_testcases_custom/ATF_Policies_External_Consent_common_functions')
+------------------------------------------------------------------------------------------------------
+---------------------------------------Preconditions--------------------------------------------------
+------------------------------------------------------------------------------------------------------
+-- Start SDL and register application
+common_functions_external_consent:PreconditonSteps("mobileConnection","mobileSession")
+-- Activate application
+common_steps:ActivateApplication("Activate_Application_1", config.application1.registerAppInterfaceParams.appName)
+------------------------------------------------------------------------------------------------------
+------------------------------------------- Functions ------------------------------------------------
+------------------------------------------------------------------------------------------------------
+local function CheckGroup001IsConsented()
+  --------------------------------------------------------------------------
+  -- Main check:
+  -- RPC of Group001 is disallowed to process.
+  --------------------------------------------------------------------------
+  Test[TEST_NAME_ON .. "MainCheck_RPC_of_Group001_is_disallowed"] = function(self)
+    local corid = self.mobileSession:SendRPC("SubscribeVehicleData",{rpm = true})
+    self.mobileSession:ExpectResponse(corid, {success = false, resultCode = "DISALLOWED"})
```

Comment:
@istoimenova, According to req-t "[Policies] External UCS: "ON" in case of omitted "disallowed_by_external_consent_entities" param in functional groupings "
SDL must process requested RPCs from "functional groupings" assigned to mobile app in terms of user_consent and data_consent policies rules
Since "rpm" is not allowed by Policies we are expecting "DISALLOWED" response